source/iterators.tex

```diff
@@ -348,11 +349,32 @@
   template<class Iterator>
     constexpr move_iterator<Iterator> operator+(
       typename move_iterator<Iterator>::difference_type n, const move_iterator<Iterator>& x);
+
   template<class Iterator>
     constexpr move_iterator<Iterator> make_move_iterator(Iterator i);
 
   template<Semiregular S> class move_sentinel;
 
+  template<class Iterator, Sentinel<Iterator> S>
+    constexpr bool operator==(
+      const move_iterator<Iterator>& x, const move_sentinel<S>& y);
+  template<class Iterator, Sentinel<Iterator> S>
+    constexpr bool operator==(
+      const move_sentinel<S>& x, const move_iterator<Iterator>& y);
+  template<class Iterator, Sentinel<Iterator> S>
+    constexpr bool operator!=(
+      const move_iterator<Iterator>& x, const move_sentinel<S>& y);
+  template<class Iterator, Sentinel<Iterator> S>
+    constexpr bool operator!=(
+      const move_sentinel<S>& x, const move_iterator<Iterator>& y);
+
+  template<class Iterator, SizedSentinel<Iterator> S>
+    constexpr iter_difference_t<Iterator> operator-(
+      const move_sentinel<S>& x, const move_iterator<Iterator>& y);
+  template<class Iterator, SizedSentinel<Iterator> S>
+    constexpr iter_difference_t<Iterator> operator-(
+      const move_iterator<Iterator>& x, const move_sentinel<S>& y);
+
```

Comment:
These operators are very much intentionally hidden friends for the effect on name lookup. This change is not editorial.

Reply:
Thank you. I guess that's the case for all the friends added?

Reply:
Yes, that is the case.